dev-tools/releng/src/main.rs

```diff
@@ -96,14 +96,15 @@ static WORKSPACE_DIR: Lazy<Utf8PathBuf> = Lazy::new(|| {
     dir
 });
 
-#[derive(Parser)]
 /// Run the Oxide release engineering process and produce a TUF repo that can be
 /// used to update a rack.
 ///
 /// For more information, see `docs/releng.adoc` in the Omicron repository.
 ///
 /// Note that `--host-dataset` and `--recovery-dataset` must be set to different
 /// values to build the two OS images in parallel. This is strongly recommended.
+#[derive(Parser)]
+#[command(name = "cargo xtask releng")]
```

Comment:
Is this here to give better error messages when this command is invoked via xtask?

It's possible that the command is still `cargo run` explicitly, instead of being invoked by xtask, right?

Reply:
I thought this adjusted the usage string because this is the same thing that's in xtask, but it actually doesn't -- this appears to just be the name of the tool, which defaults to the crate name. So I'm going to try and find the correct thing and adjust that here and in xtask.